BUILD.md

```diff
@@ -135,6 +135,12 @@ cd cactus
 yarn run configure
 ```
 
+* Run to enable corepack which helps with managing versions of your package managers.
+
+```sh
+npm run corepack
+```
+
```

Comment:
@charellesandig I would move this up to be just above the `configure` script so that the corepack enablement happens first, which pins the yarn version to the preferred one and THEN the configure script is executed with that desired yarn version as defined by the corepack enablement script.